wasm/src/lib.rs

```diff
@@ -179,20 +179,40 @@ pub fn query_triggering_event() -> Event {
     unsafe { decode_with_length_prefix_from_raw(host_query_triggering_event()) }
 }
 
-/// Query an operation which is to be validated
+/// Query [`VersionedSignedTransaction`] to validate.
```

Comment:
Why is `query` used for these functions instead of something like `get`? It feels as if it's in some way related to Iroha `Query`

Reply:
I'm not particularly on `query_` side, but it's named so for historical reasons, I believe. Need other opinions here.
@mversic , @Erigara  ?